MPM/mpm_settings.php

```diff
@@ -333,6 +333,67 @@ public function gateways_add_static($gateways)
 		$head = array_slice($gateways, 0, $pos);
 		$tail = array_slice($gateways, $pos);
 		$gateways = array_merge($head, $mollie_gateways, $tail);
+		return $this->reorderGateways($gateways);
+	}
+
+	/**
+	 * Reorders the method listing on the front-end.
+	 * For special countries, such as Belgium should order MisterCash as first method
+	 * @param array $gateways
+	 * @return array $gateways
+	 */
+	public function reorderGateways($gateways)
```

Comment:
Mollie sorteert de betaalmethodes zelf ook al aan de hand van de locale. https://github.com/vanengers/WooCommerce/blob/master/MPM%2Fmpm_settings.php#L523 stuurt sinds kort ook de locale mee. Bij 'be', 'nl_BE' of 'fr_BE' is de volgorde bijvoorbeeld:
creditcard, mistercash, belfius, paypal, sofort, ideal, banktransfer, bitcoin, paysafecard

Ik vraag mij alleen af of er in WordPress een specifieke Belgische locale is.

Ik zie in de methode geen volgorde per locale / land. Wordt de volgorde bijvoorbeeld voor Nederlandse klanten nu ook veranderd? Verder is deze code lastig te begrijpen. Is deze methode echt nodig? Is dit voor een specifieke Belgische klant?

In versie 2.x zijn alle gateways zelf te orderen via de standaard WooCommerce functionaliteit. Dit zal dus een tijdelijke fix zijn voor 1.x